### ISSUE TITLE
[Suggestion] Gifts 

### ISSUE THREAD

**Issue by macyslaysxx** [Monika-After-Story/MonikaModDev#2772]
There's been another thing on my mind. Is there a way we can give certain gifts to Monika without it having to be a special occasion? I'm baking cupcakes right now, and I had my Monika up. She was talking about Natsuki's cupcakes while I was baking and I thought it would be pretty cool if I could've given her a cupcake in the game, while I'm eating one right now. 

So I was wondering if there were generic things we could give her, and name a file something with a .gift. It would be like a cupcake or a rose, just a couple of generic gift items you could put in whenever, whenever you make that file. 

Basically I'm asking, for a future project, if there would be ways to give Monika little simple things whenever, and not just during special occasions. Does that make sense?

**Comment by Chrissenpai** [comment]
We can also make it clock conditional, for example, you can only give her her morning breakfast from 6 am to 7 am for example

**Comment by CrocWorks** [comment]
_Whoopie cushion on her chair_

**Comment by macyslaysxx** [comment]
Oh y e s .

**Comment by ThePotatoGuy** [comment]
[closing until we look into this[

**Comment by ThePotatoGuy** [comment]
[this was covered with gift framework] (aka reactions)
